component/thanos.libsonnet

```diff
@@ -0,0 +1,95 @@
+local com = import 'lib/commodore.libjsonnet';
+local kap = import 'lib/kapitan.libjsonnet';
+local kube = import 'lib/kube.libjsonnet';
+
+local thanosMixin = import 'github.com/thanos-io/thanos/mixin/mixin.libsonnet';
+local thanos = import 'kube-thanos/thanos.libsonnet';
+
+local inv = kap.inventory();
+local params = inv.parameters.openshift4_monitoring;
+
+local receiverCfg = {
+  local this = self,
+  name: 'thanos-receive',
+  image: '%(registry)s/%(repository)s:%(tag)s' % params.images.thanos,
+  version: params.images.thanos.tag,
+  serviceMonitor: true,
+  retention: '1d',
+  objectStorageConfig: {
+    name: this.name + '-objstorage',
+    key: 'thanos.yaml',
+  },
+  resources: {
+    requests: {
+      cpu: '500m',
+      memory: '4Gi',
+    },
+    limits: {
+      cpu: '4',
+      memory: '10Gi',
+    },
+  },
```

Comment:
At the moment, it's not clear to me whether these requests and limits are actually sensible.